trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -314,60 +333,46 @@ describe(page, () => {
 
-  it('should render the utilisation tab table', () => {
-    const feeRecordId = 12;
-    const wrapper = getWrapper({
-      ...params,
-      utilisationDetails: {
-        utilisationTableRows: [
-          {
-            ...aUtilisationTableRowViewModel(),
-            feeRecordId,
-          },
-        ],
-        downloadUrl,
-      },
+      wrapper.expectText(`${utilisationTabSelector} h2[data-cy="bank-report-heading"]`).toRead('Bank report');
     });
 
-    const utilisationTabSelector = 'div#utilisation';
+    it('should render the utilisation tab table', () => {
+      const feeRecordId = 12;
+      const wrapper = getWrapper({
+        ...params,
+        utilisationDetails: {
+          utilisationTableRows: [
+            {
+              ...aUtilisationTableRowViewModel(),
+              feeRecordId,
+            },
+          ],
+          downloadUrl,
+        },
+      });
 
-    wrapper.expectElement(`${utilisationTabSelector} table`).toExist();
-    wrapper.expectElement(`${utilisationTabSelector} table tr[data-cy="utilisation-table-row-${feeRecordId}"]`);
-  });
+      const utilisationTabSelector = 'div#utilisation';
 
-  it('should render the download link for the report', () => {
-    const wrapper = getWrapper({
-      ...params,
-      utilisationDetails: {
-        utilisationTableRows: [aUtilisationTableRowViewModel()],
-        downloadUrl,
-      },
+      wrapper.expectElement(`${utilisationTabSelector} table`).toExist();
+      wrapper.expectElement(`${utilisationTabSelector} table tr[data-cy="utilisation-table-row-${feeRecordId}"]`);
     });
 
-    wrapper.expectLink('[data-cy="download-report-link"]').toLinkTo(downloadUrl, 'Download the report submitted by the bank as a CSV');
-  });
-
-  it('should not display match success notification when param is false', () => {
-    const wrapper = getWrapper({ ...params, displayMatchSuccessNotification: false });
-
-    wrapper.expectElement('[data-cy="match-success-notification"]').notToExist();
-  });
-
-  it('should display match success notification when param is true', () => {
-    const wrapper = getWrapper({ ...params, displayMatchSuccessNotification: true });
+    it('should render the download link for the report', () => {
+      const wrapper = getWrapper({
+        ...params,
+        utilisationDetails: {
+          utilisationTableRows: [aUtilisationTableRowViewModel()],
+          downloadUrl,
+        },
+      });
 
-    wrapper.expectElement('[data-cy="match-success-notification"]').toExist();
-    wrapper.expectText('[data-cy="match-success-notification-heading"]').toRead('Match payment recorded');
-    wrapper
-      .expectText('[data-cy="match-success-notification-message"]')
-      .toRead('The fee(s) are now at a Match state. Further payments cannot be added to the fee record.');
+      wrapper.expectLink('[data-cy="download-report-link"]').toLinkTo(downloadUrl, 'Download the report submitted by the bank as a CSV');
+    });
```

Comment:
Sorry had to comment here as it's not possible to comment on an image file - it looks like `icon-tag-remove-cross.svg` has been added but is not being used, unless i'm missing something?

Reply:
It is used by the moj filter component, without it the selected filters have no crosses (see screenshots in PR description to see cross showing up)